Aspire BudgetingTests/InstabugCredentialsTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  InstabugCredentialsTests.swift
+//  Aspire BudgetingTests
+//
+//  Created by TeraMo Labs on 3/6/20.
+//  Copyright © 2020 TeraMo Labs. All rights reserved.
+//
+
+@testable import Aspire_Budgeting
+import XCTest
+
+class InstabugCredentialsTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/f746b455a622f9990e873788abf8097326f10d22/Aspire BudgetingTests/InstabugCredentialsTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>